CPP11.md

```diff
@@ -738,6 +739,27 @@ C++11 introduces a memory model for C++, which means library support for threadi
 
 See the sections on: [std::thread](#stdthread)
 
+### std::async
+`std::async` runs function asynchronously or perform lazy evaluation, then returns a `std::future` which holds the result of that function call.
```

Comment:
`runs the given function either asynchronously or lazily-evaluated`*
Just some minor tweaks here.